x/ecocredit/client/util.go

```diff
@@ -1,36 +1,54 @@
 package client
 
 import (
-	"github.com/cosmos/cosmos-sdk/client"
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/tx"
 	"github.com/gogo/protobuf/proto"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
+	"github.com/regen-network/regen-ledger/client"
 	"github.com/regen-network/regen-ledger/x/ecocredit"
 )
 
-func print(ctx client.Context, res proto.Message, err error) error {
+// prints a query client response
+func print(cctx sdkclient.Context, res proto.Message, err error) error {
 	if err != nil {
 		return err
 	}
-	return ctx.PrintOutput(res)
+	return cctx.PrintOutput(res)
 }
 
-func mkTx(ctx client.Context, cmd *cobra.Command, err error) error {
-	if err != nil {
-		return err
-	}
-	return tx.GenerateOrBroadcastTxCLI(ctx, cmd.Flags())
+func mkQueryClient(cmd *cobra.Command) (ecocredit.QueryClient, sdkclient.Context, error) {
+	ctx := sdkclient.GetClientContextFromCmd(cmd)
+	ctx, err := sdkclient.ReadQueryCommandFlags(ctx, cmd.Flags())
+	return ecocredit.NewQueryClient(ctx), ctx, err
 }
 
-func mkQueryClient(cmd *cobra.Command) (ecocredit.QueryClient, client.Context, error) {
-	ctx := client.GetClientContextFromCmd(cmd)
-	ctx, err := client.ReadQueryCommandFlags(ctx, cmd.Flags())
-	return ecocredit.NewQueryClient(ctx), ctx, err
+type msgSrvClient struct {
+	Cctx   *sdkclient.Context
```

Comment:
```suggestion
	ClientContext   *sdkclient.Context
```

Reply:
The type name already has the full meaning: `ClientContext sdk.clientContext`. I can change to `ClientCtx` if you prefer.